src/Tickets/Commerce/Admin_Tables/Orders_Table.php

```diff
@@ -449,6 +449,10 @@ public function column_items( $item ) {
 		}
 
 		foreach ( $item->items as $cart_item ) {
+			// Check if 'type' exists and proceed only if it's empty or equals 'ticket'.
+			if ( ! empty( $cart_item['type'] ) && 'ticket' !== $cart_item['type'] ) {
+				continue;
+			}
```

Comment:
```suggestion
			if ( empty( $cart_item['type'] ) || 'ticket' !== $cart_item['type'] ) {
				continue;
			}
```

Lets assure this is backwards compatbile. The typ;e is being added now, so for already existing entries its not expected to be in place. For those it should be assume the item is a ticket